.coveragerc

```diff
@@ -390,6 +390,9 @@ omit =
     homeassistant/components/goalzero/binary_sensor.py
     homeassistant/components/goalzero/sensor.py
     homeassistant/components/goalzero/switch.py
+    homeassistant/components/goodwe/__init__.py
+    homeassistant/components/goodwe/const.py
+    homeassistant/components/goodwe/sensor.py
```

Comment:
It is not compulsory, but since new integrations can take a while to get merged, I suggest that you take the opportunity to move to 100% coverage whilst the reviews are in progress.